recipes/gfortran_osx-64/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set gfortran_version = [7, 3, 0] %}
+{% set libgfortran_version = [4, 0, 0] %}
+{% set max_libgfortran_version = "5.0.0.a0" %}
+{% set chost = "x86_64-apple-darwin13.4.0" %}
+
+package:
+  name: gfortran_osx-64
+  version: {{ gfortran_version|join('.') }}
+
+build:
+  number: 0
+  skip: True  # [not osx]
+  binary_has_prefix_files:
+    - lib/libgcc_ext.10.*.dylib
+  always_include_files:
+    - bin/gfortran
+    - bin/{{ chost }}-gfortran
+
+    - libexec/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/collect2
+    - libexec/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/f951
+    - libexec/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/lto-wrapper
+
+    - lib/libgfortran.spec
+
+    # For -fopenmp:
+    - lib/libgomp.spec
+
+    # For -ffast-math
+    - lib/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/crtfastmath.o
+
+    # For -static:
+    - lib/libgfortran.a
+    - lib/libgomp.a
+
+    - lib/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/libgcc.a
+    - lib/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/libgcc_eh.a
+
+    - lib/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/finclude/**
+
+    # include{,-fixed} may not be needed unless -fopenmp is passed (not sure on that):
+    - lib/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/include-fixed/**
+    - lib/gcc/{{ chost }}/{{ gfortran_version|join('.') }}/include/**
+
+    # Stub libraries:
+    - lib/libgcc_ext.10.4.dylib
+    - lib/libgcc_ext.10.5.dylib
```

Comment:
`gfortran_osx-64` should just depend on `gfortran_impl_osx-64` at runtime. This set of files should be moved to `gfortran_osx-64` as the set of files as in https://conda.io/projects/conda-build/en/latest/source/define-metadata.html#specifying-files-to-include-in-output

Reply:
Do you mean only the dylibs, or do we want gfortran_osx-64 to be basically a shell package?

Reply:
I meant all files. We want `gfortran_osx-64` to be a package with just activation scripts like `gcc_osx-64`